woocommerce/woocommerce
# Fix add missing dev package

### All Submissions:






### Changes proposed in this Pull Request:

This should fix a broken build caused by pnpm version 7.29.2



### How to test the changes in this Pull Request:




1. Make sure Github actions run successfully
2. Install pnpm `7.29.2` locally and make sure the build runs successfully: `TURBO_FORCE=true pnpm run build`



### Other information:




### FOR PR REVIEWER ONLY:
